backend/services/appointmentDeletionService.js

```diff
@@ -0,0 +1,80 @@
+const moment = require('moment')
```

Comment:
Not sure on the filename.  Feels more like `appointmentDetailsService.js` would be more suitable as there's nothing related to deletion in this file.